documentation/src/physical_types/mixed_systems.md

```diff
@@ -109,16 +109,16 @@ let mdp = MixedDecoherenceProduct::new([dp], [bp], [fp]).unwrap();
 Complex objects are constructed from operator products are `MixedOperators` and `MixedHamiltonians`
 (for more information, [see also](../container_types/operators_hamiltonians_and_systems.md)).
 
-These `MixedOperators` and `MixedHamiltonians` represent operators or hamiltonians such as:
-\\[ \hat{H} = \sum_{j=0}^N \alpha_j \prod_{k} \sigma_j^{k}  \prod_{l, m} c_{b, l, j}^{\dagger} c_{b, m, j} \prod_{r, s} c_{f, r, j}^{\dagger} c_{f, s, j} \\]
+These `MixedOperators` and `MixedHamiltonians` represent operators or Hamiltonians such as:
+\\[ \hat{H} = \sum_j \alpha_j \prod_k \sigma_{j, k}^{a} \prod_{l, m} c_{b, l, j}^{\dagger} c_{b, m, j} \prod_{r, s} c_{f, r, j}^{\dagger} c_{f, s, j} \\]
```

Comment:
What is the new exponent/superscript `a` here?

Reply:
A mistake, thank you! Removed